src/addremoveedit.js

```diff
@@ -0,0 +1,61 @@
+import { save, load } from './data.js';
+
+const theBigList = document.querySelector('.lists');
+
+export function ShowAll(element) {
+  while (element.lastElementChild) {
+    element.removeChild(element.lastElementChild);
+  }
+}
+
+export function removeCompleteds() {
+  const actualtodo = load();
+  const result = actualtodo.filter((activity) => activity.completed === false);
+  for (let i = 0; i < result.length; i += 1) {
+    result[i].index = i;
+  }
+  save(result);
+}
+
+export function addActivity(text) {
+  const actualtodo = load();
+  const { length } = actualtodo;
+  const newActivity = {
+    description: text,
+    completed: false,
+    id: Date(),
+    index: length,
+  };
+  actualtodo[length] = newActivity;
+  save(actualtodo);
+}
+
+export function eliminateOne(element) {
+  // eliminate one
+  const { id } = element;
+  const actualtodo = load();
+  const result = actualtodo.filter((activity) => activity.id === id);
+  element.parentNode.removeChild(element);
+  const e = theBigList.querySelectorAll('.list');
+  for (let i = 0; i < result.length; i += 1) {
+    result[i].index = i;
+    e[i].id = i;
+  }
+  save(result);
+}
```

Comment:
- [ ] Kindly make sure that your items are added to the list and deleted when you click "clear all completed" button after checking it.
- [ ] Kindly also make sure that when you mark the check box there should be a line through the text.

_Right now the items are not added nor deleted. The below video is your app. kindly fix it_

https://user-images.githubusercontent.com/70269507/173155033-493fa8a4-7009-4058-b009-c7683a45d88e.mov

You can watch this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska) to have a look at how items are added, edited and deleted.

Reply:
@sja-thedude Hi thank you for your review and making me a better developer 💛 

Happy coding!!